frontend/src/queries/nodes/DataTable/DataTable.tsx

```diff
@@ -65,8 +65,8 @@ let uniqueNode = 0
 
 export function DataTable({ uniqueKey, query, setQuery, context, cachedResults }: DataTableProps): JSX.Element {
     const uniqueNodeKey = useState(() => uniqueNode++)
-    const [vizKey] = useState(() => `DataTable.${uniqueKey || uniqueNodeKey}`)
     const [dataKey] = useState(() => `DataNode.${uniqueKey || uniqueNodeKey}`)
+    const [vizKey] = useState(() => `DataTable.${uniqueNodeKey}`)
```

Comment:
Needed to revert to an [earlier commit](https://github.com/PostHog/posthog/pull/17187/commits/fd2a4cd39a28dd8364416184919e014cc3baec5d) because using the same key for the viz & data meant that both representations shared view props e.g. the table was showing up in the settings panel